include/openssl/conf.h

```diff
@@ -34,8 +34,13 @@ typedef struct {
 } CONF_VALUE;
 
 DEFINE_OR_DECLARE_STACK_OF(CONF_VALUE)
-
+DEFINE_OR_DECLARE_STACK_OF(CONF_MODULE)
```

Comment:
The `typedef struct conf_module_st CONF_MODULE;` around line 64 below needs to occur before this line. I was getting compile errors in my application including the conf.h file without this.

Reply:
moved the macro after the typedef, thanks.